docs/avdanos/contributing/design/OXYDE/main-style.md

```diff
@@ -0,0 +1,121 @@
+---
+sidebar_position: 2
+title: Main style
+description: The main style of our design.
+---
+
+:::caution
+Reminder: OXYDE stills in Beta phase.
+:::
+
+# Fonts
+
+When designing, fonts are an important point. AvdanOS font for User Interface is **Ubuntu**.
+![OXYDE Fonts](Docsite/static/img/Fonts.png)
+
+# Colors
+
+Other imporant point is adding color to our program. Designing with consisten colors makes the experience more intuitive, by making the UI feel more alive and visual. So, these are the AvdanOS main colors, both for Dark mode and Light mode.
+:::tip
+Although there are many themes in AvdanOS using slightly different colors, each theme has a tag that defines if it is a light theme or dark theme, based on the main and accent colors.
+:::
+![OXYDE Colors](Docsite/static/img/Colors.png)

```

Comment:
For absolutely no reason there is a compiling error:
![image](https://user-images.githubusercontent.com/87482502/216966617-cfd1ee3a-1dfa-428c-b859-d982b7d4ac79.png)

Fix:
Just Remove `Docsite/static` In All Of the places

```suggestion
![OXYDE Colors](/img/Colors.png)

```